molecule/default/verify.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Verify role
+  hosts: all
+  become: yes
+
+  vars:
+    drupal_log_path: /var/log/vhosts/account/project/drupal.log
+    drupal_syslog_read_users:
+      - johndoe
+
+  tasks:
+    - name: Obtain the ACL for drupal log
+      acl:
+        path: "{{ drupal_log_path }}"
+      register: acl_info
+
+    - debug: var=acl_info
```

Comment:
While it's useful to be able to examine info to help debug problems, "verify" tasks should programmatically return a pass fail: exit cleanly or exit with error. This looks like it will always pass.